package.json

```diff
@@ -33,13 +33,16 @@
   "dependencies": {
     "@caporal/core": "2.0.2",
     "@mdn/browser-compat-data": "2.0.7",
+    "accept-language-parser": "1.5.0",
```

Comment:
`sanitize-filename` was already part of this `package.json`. So now we have the same dependencies installed that we have in `origin-request`.